modules/adfs/www/idp/prp.php

```diff
@@ -23,6 +23,6 @@
 	// logout response from ADFS SP
 	$assocId = $_GET['assocId']; // Association ID of the SP that sent the logout response
 	$relayState = $_GET['relayState']; // Data that was sent in the logout request to the SP. Can be null
-	$logoutError = NULL; /* NULL on success, or an instance of a SimpleSAML_Error_Exception on failure. */
+	$logoutError = null; /* NULL on success, or an instance of a SimpleSAML_Error_Exception on failure. */
```

Comment:
I would update the comment too (s/NULL/null/). In general, I also prefer single-line comment markers (`//`) for single-line comments, and multi-line comment markers (`/* */`) for multi-line comments, but that's more personal preference and you don't have to do that if you don't want to 😄